### ISSUE TITLE
dynamodbstreams records do not contain ApproximateCreationDateTime

### ISSUE THREAD

**Issue by Neos3452** [localstack/localstack#2780]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

First of all, thanks a lot for making such great tool, it is really making local development with aws much easier!

The problem is that when GetRecords request is sent https://docs.aws.amazon.com/cli/latest/reference/dynamodbstreams/get-records.html ApproximateCreationDateTime does not appear in the response. Unfortunately this causes some libraries that provide kinesis<->dynamodbstreams compatibility layer crash.

I took a look at the dynamodb `localstack/services/dynamodb/dynamodb_listener.py` where my understanding is that everything is happening, and it seems that ApproximateCreationDateTime simply is not added to a record. It seems that localstack returns eventVersion 1.0 while AWS is already on 1.1.

## Expected behavior

```json
{
    "Records": [
        {
            "eventID": "1",
            "eventName": "INSERT",
            "eventVersion": "1.0",
            "eventSource": "aws:dynamodb",
            "awsRegion": "us-east-1",
            "dynamodb": {
                "ApproximateCreationDateTime": 1571849028.0,
                "Keys": {
                    "Name": {
                        "S": "test-00"
                    }
                },
                "NewImage": {
                    "Name": {
                        "S": "test-00"
                    }
                },
                "SequenceNumber": "1",
                "SizeBytes": 26,
                "StreamViewType": "NEW_AND_OLD_IMAGES"
            }
        }
    ],
    "NextShardIterator": "AAAAAAAAAAE9j4BP9rsMxqJqUfrLoMNgr/VWKIbhqU4w/nV2vHvSilm6Hvg7VSnbR9g8EDcYaskjMzE2VxUWof2/miEKCae9Itikem8RoLH9fCihVIm+keRToMGjN0BSyQlgfgMwvKKjuCg/2G9nYtnKHJgmUSB5tNuZYtrVMfqonLADm9rvWet1QLL7Kv+ECGwORgp/tMRf42So+TynwGkNx4apsRlY"
}

```

## Actual behavior

```json
{
    "Records": [
        {
            "eventID": "1",
            "eventName": "INSERT",
            "eventVersion": "1.0",
            "eventSource": "aws:dynamodb",
            "awsRegion": "us-east-1",
            "dynamodb": {
                "Keys": {
                    "Name": {
                        "S": "test-00"
                    }
                },
                "NewImage": {
                    "Name": {
                        "S": "test-00"
                    }
                },
                "SequenceNumber": "1",
                "SizeBytes": 26,
                "StreamViewType": "NEW_AND_OLD_IMAGES"
            }
        }
    ],
    "NextShardIterator": "AAAAAAAAAAE9j4BP9rsMxqJqUfrLoMNgr/VWKIbhqU4w/nV2vHvSilm6Hvg7VSnbR9g8EDcYaskjMzE2VxUWof2/miEKCae9Itikem8RoLH9fCihVIm+keRToMGjN0BSyQlgfgMwvKKjuCg/2G9nYtnKHJgmUSB5tNuZYtrVMfqonLADm9rvWet1QLL7Kv+ECGwORgp/tMRf42So+TynwGkNx4apsRlY"
}

```

# Steps to reproduce

## Command used to start LocalStack

Via docker:

```dockerfile
FROM localstack/localstack:latest

ENV DOCKER_HOST=unix:///var/run/docker.sock \
    SERVICES=kinesis,dynamodb,dynamodbstreams,sns,sqs,s3 \
    DATA_DIR=/tmp/data \
    USE_SSL=true \
    START_WEB=0

EXPOSE 4566-4597 8080
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

```bash
awslocal dynamodb create-table --table-name LocalstackTesst --attribute-definitions AttributeName=Name,AttributeType=S --key-schema AttributeName=Name,KeyType=HASH --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --stream-specification StreamEnabled=true,StreamViewType=NEW_AND_OLD_IMAGES
awslocal dynamodb put-item --table-name LocalstackTesst --item '{"Name":{"S":"test-00"}}'
awslocal dynamodb describe-table --table-name LocalstackTesst # get arn
awslocal dynamodbstreams describe-stream --stream-arn "arn:aws:dynamodb:us-east-1:000000000000:table/LocalstackTesst/stream/2020-07-28T10:30:57.768" # get shardid
awslocal dynamodbstreams get-shard-iterator --stream-arn "arn:aws:dynamodb:us-east-1:000000000000:table/LocalstackTesst/stream/2020-07-28T10:30:57.768" --shard-id "shardId-00000001595900000000-000000000000" --shard-iterator-type TRIM_HORIZON # get iterator
awslocal dynamodbstreams get-records --shard-iterator "AAAAAAAAAAGF83o8gcnTSbFvBbOlzOFEcKJhGr+DYc0dpPUIFdC171s6pP4AtehVRo5ALNIOPhCAK4vWW37EUQ0qHnk6b/Ke8cGYVwftmB1xMXqUjD15CTW0gMI2VyVumFevgHvM0emsfJ/bRVDKiTKWhfiPdW8o5k4JiCdLzpl4FyMqJa6jQf/q7HFSYaHHZwdfFd2SCBjnV7ziWQqGpd5vsi0qjR3a"
```

**Comment by whummer** [comment]
Thanks for the detailed report @Neos3452 . This should be fixed in #2800 - can you please do a `docker pull localstack/localstack` and give it another try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks

**Comment by Neos3452** [comment]
Thanks a lot, works like a charm!
